tests/integration/view/one_to_many/simple_test.go

```diff
@@ -239,3 +239,67 @@ func TestView_OneToManyOuterToInnerToOuter_Errors(t *testing.T) {
 
 	testUtils.ExecuteTestCase(t, test)
 }
+
+func TestView_OneToManyWithRelationInQueryButNotInSDL(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "One to many view",
```

Comment:
todo: please the description more descriptive :)

Reply:
Lol - sorry :grin: 

- [ ] Sort out description(s?)